content/docs/user-guide/external-dependencies.md

```diff
@@ -27,20 +28,19 @@ supported:
 - HTTP
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
-
-In order to specify an external <abbr>dependency</abbr> for your stage, use the
-usual `-d` option in `dvc run` with the external path or URL to your desired
-file or directory.
+> Note [remote storage](/doc/command-reference/remote) is a separate feature.
 
 ## Examples
 
-Let's take a look at a `download_file` [stage](/doc/command-reference/run) that
-simply downloads a file from an external location.
+To define an external <abbr>dependency</abbr>, add the external URL or path to
```

Comment:
let's simplify this ... reading two paragraphs of text as intro to an example looks wrong

Reply:
I think it was fine to keep in the previous paragraph?

Reply:
> it was fine to keep in the previous paragraph

Yes, going back to that (almost) now that the mechanism is explained a little more in the previous section.

Done in de67413d